### ISSUE TITLE
Asking for permission

### ISSUE THREAD

**Issue by why-trv** [jatalahd/tsc#9]
Hi! 

I've been using TSC in the web occasionally over the years and loving it. Really appreciate having a platform-independent version since I rarely use Windows. That said, I find some features missing. For instance, there's no built-in way to compare different tonestack topologies, the workflow for editing values could be faster, etc. Thinking that would be too much to ask with the current structure of the project, I decided to have a go at hacking together my own implementation.

This is what it looks like at the moment:
<img width="1522" alt="Screenshot 2024-09-12 at 18 20 21" src="https://github.com/user-attachments/assets/cd021fd5-773c-4ec0-be4d-ebcffc916d3d">

Would you mind if I copy the math from your project over to mine and publish my calculator?

(As you probably can see, you can add multiple tonestacks and tweak them after the fact, also there is an option to use 'global' sliders on all tonestacks simultaneously and see how they respond. The part value inputs allow manual input as well as going to next or previous standard value (E24) using the +- buttons or mouse scroll, etc.)

**Comment by jatalahd** [comment]
Hello,

Looks very nice. Of course you can use the maths from this project. It is not protected by any license or anything like that. Free to use for everyone.

**Comment by why-trv** [comment]
Thanks! Even though it's technically free, there's been a lot of work put into it, that's why I asked :)

My calculator is now live here: https://tonestack.yuriturov.com
And this is the repo: https://github.com/why-trv/yet-another-tonestack-calculator
I hope you like it

**Comment by jatalahd** [comment]
Yes, it is excellent. There are many features added which had been requested over the years, but never implemented here. As there has not been much activity in this repo for more than a year, I will retire this project and hand you the torch to maintain the better version. I will keep this version still online for historical reasons, as it is on my personal web page.
